packages/embedded/src/test/kotlin/elide/embedded/NativeEmbeddedRuntimeTest.kt

```diff
@@ -48,7 +47,7 @@ class NativeEmbeddedRuntimeTest {
   @TempDir lateinit var testGuestRoot: Path
 
   /** Create a GraalVM native isolate thread in the current arena and return a pointer to it. */
-  context(Arena) private fun prepareIsolate(): MemorySegment {
+  private fun Arena.prepareIsolate(): MemorySegment {
```

Comment:
@darvld had trouble with `context(...)`, so these are now within the context of `Arena`; the trouble is probably related to JDK22, in that `allocateStringUtf8` and `allocateArray` have been yeeted. as a result, for now, `embedded` is pinned to JDK21.